contracts/.solhint-test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "extends": "solhint:recommended",
+  "plugins": ["prettier", "chainlink-solidity"],
+  "rules": {
+    "compiler-version": ["off", "^0.8.0"],
+    "const-name-snakecase": "off",
+    "constructor-syntax": "error",
+    "var-name-mixedcase": "off",
+    "func-named-parameters": "off",
+    "immutable-vars-naming": "off",
+    "no-inline-assembly": "off",
+    "contract-name-camelcase": "off",
+    "one-contract-per-file": "off",
+    "avoid-low-level-calls": "off",
```

Comment:
Mostly the same but allows 

- avoid-low-level-calls
- one-contract-per-file
- contract-name-camelcase